### ISSUE TITLE
How can I provide the value for what's new?

### ISSUE THREAD

**Issue by NPadrutt** [microsoft/app-store-vsts-extension#179]
## Note  
Issues in this repo are for tracking bugs, feature requests and questions for the tasks in this repo 

For a list:  
https://github.com/microsoft/app-store-vsts-extension/tree/master/Tasks

If you have an issue or request for the Azure Pipelines service, use developer community instead:  

https://developercommunity.visualstudio.com/spaces/21/index.html )

## Required Information

Entering this information will route you directly to the right team and expedite traction.

**Question, Bug, or Feature?**  
*Type*: Question

**Enter Task Name**: Apple App Store Release

list here (V# not needed):  
https://github.com/microsoft/app-store-vsts-extension/tree/master/Tasks

## Environment
- Server - Azure Pipelines or TFS on-premises?
    
    - If using TFS on-premises, provide the version: 
    
    - If using Azure Pipelines, provide the account name, team project name, build definition name/build number: 
https://dev.azure.com/applysolutions/MoneyFox/_releaseProgress?_a=release-pipeline-progress&releaseId=255

- Agent - Hosted or Private: 
    
    - If using Hosted agent, provide agent queue name:

    - If using private agent, provide the OS of the machine running the agent and the agent version: 

## Issue Description
I try to submit a build I uploaded to a testflight to production in the apple store. The one thing I haven't figured out how to do is 
provide "what's new" field. I checked the fastlane documentation and from what I see you should be able to provide a release note via the meta data. This is the file structure I used:

![image](https://user-images.githubusercontent.com/1764367/95677058-3aebeb00-0bc3-11eb-8a53-42a65c2376d6.png)

I also tried it, when I move the release notes down into a default folder, in order for fast lane to use this as the fallback language. but this didn't work.
How would I have to provide those information?

### Task logs

[Enable debug logging and please provide the zip file containing all the logs for a speedy resolution]

## Troubleshooting
Checkout how to troubleshoot failures and collect debug logs: https://docs.microsoft.com/en-us/vsts/build-release/actions/troubleshooting

### Error logs

[Insert error from the logs here for a quick overview]

**Comment by anatolybolshakov** [comment]
Hi @NPadrutt it seems to be some external issue with fastlane - could you please check it here? https://github.com/fastlane/fastlane/issues/16700
According to comment here in thread - could you please check the role also?
https://github.com/fastlane/fastlane/issues/16700#issuecomment-706710412

**Comment by NPadrutt** [comment]
@anatolybolshakov  Thanks for your reply and the link to the issue. The user I use is admin so permissions shouldn't be an issue. I'll try a couple of the other suggestions from that thread and will let you know how that went :)

**Comment by NPadrutt** [comment]
@anatolybolshakov so the issue was, that I haven't passed a list of languages and with that the fallback doesn't work. Unfortunately I haven't found out how to pass that list in Azure Releas Pipeline. As a work around I just create a folder for every language.
In case you have a hint how I can pass that via the additional fastlane arguments that would be awesome though :)

**Comment by anatolybolshakov** [comment]
@NPadrutt ok, thanks! Could you try to use '--languages "<List of activated languages>"' value for additional fastlane arguments according to https://docs.fastlane.tools/actions/deliver/? Looks like this should do the trick here

**Comment by anatolybolshakov** [comment]
Hi @NPadrutt  is this issue still actual for you?

**Comment by NPadrutt** [comment]
@anatolybolshakov sorry for not getting back to you. I tried several variations with the languages parameter, but so far wasn't able to make it work. I'm not sure if I have to pass all the language in a specific way in the parameter. For example I tried:

`--languages "de,en,fin"
`
or

`
--languages "[de,en,fin]"
`

or with escaped high commas.

`--languages "[\"de\",\"en\",\"fin\""`

But I mean, for me it works with creating empty folders for each language via a powershell script. Although not elegant, it does the job.
